src/utils/framework.ts

```diff
@@ -380,6 +376,7 @@ export const getMatrix1dToc = (
                     id: cellKey,
                     key,
                     title: value,
+                    altTitle: `${title} > ${rowTitle} > ${value}`,
```

Comment:
We should use the chevron

Reply:
Maybe create a function
```javascript
function breadcrumb(...args: string[]) {
    return args.join(' › ');
}
```
We can make this a fancier function with:
1. default names for undefined values
2. prop to configure separator
3. filter out undefind/null values
etc.